src/main/java/org/sonarlint/intellij/actions/SonarLintToolWindow.java

```diff
@@ -105,7 +100,10 @@ public void openCurrentFileTab() {
   }
 
   public void openSecurityHotspotsTab() {
-    openTab(getSecurityHotspotContent());
+    var content = getSecurityHotspotContent();
+    if (content != null) {
```

Comment:
In which case would this happen to be `null`? I am a bit worried about not being able to open the tab when requested.

Reply:
Right, this shouldn't be the case here, I will remove it

It is needed on the method called during the on the fly analysis, it seems it can be executed before the hotspot tab is rendered (at least it makes the tests fail)